### ISSUE TITLE
Amazon error: Need to first confirm Terms and Conditions

### ISSUE THREAD

**Issue by DonSchenck** [redhat-developer-demos/openshift-hybridizer#8]
My Amazon setup is new, so I received this error when running provision.sh:
```
TASK [cloud-resources : AWS Create All in one OpenShift Node] ******************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Instance creation failed => OptInRequired: In order to use this AWS Marketplace product you need to accept terms and subscribe. To do so please visit https://aws.amazon.com/marketplace/pp?sku=aw0evgkw8e5c1q413zgy5pjce"}
```

**Comment by DonSchenck** [comment]
Need to approve the billing rate of 1.2 cents per hour for CentOS.

**Comment by DonSchenck** [comment]
Also; AFTER accepting said billing, you need to wait a bit or you'll get the following error:
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Instance creation failed => PendingVerification: Your account is currently being verified. Verification normally takes less than 2 hours. Until your account is verified, you may not be able to launch additional instances or create additional volumes. If you are still receiving this message after more than 2 hours, please let us know by writing to aws-verification@amazon.com. We appreciate your patience."}
```

**Comment by rafabene** [comment]
This should be solved with #14
